### ISSUE TITLE
Adaptive mode override

### ISSUE THREAD

**Issue by TizianoNoto** [AntonyCorbett/OnlyT#388]
**Is your feature request related to a problem? Please describe.**
I tried to set the congregation bible study to 20 minutes. Since the meeting would have ended 7 minutes late, OnlyT still chose and displayed 13 minutes from the adapted time.

Disabling the adaptive mode didn't help until I restarted the application (and lost the report).

**Describe the solution you'd like**
I'd like to enable the operator to override the adaptive mode if needed.

**Describe alternatives you've considered**
Force the speakers to end on time, no matter what. 😄 

**Additional context**
It was the first meeting with a videoconference system and some parts lasted way longer than expected.

For time management we chose OnlyT, enabled the web clock and set port forwarding on my router thus allowing the speakers to connect to it.

**Comment by AntonyCorbett** [comment]
@TizianoNoto The adaptive timing can be disabled without having to close OnlyT. If you can reproduce the error please let me know. Thanks! Antony

**Comment by TizianoNoto** [comment]
I forgot to mention I'm running version 1.3.0.1.
Anyway, I think I found what the problem was.

When starting the timer with one-way adaptive mode enabled the application will show the user it picks the shortest duration:
![immagine](https://user-images.githubusercontent.com/42154401/77684715-9d202e00-6f9a-11ea-9b8a-5640898bf703.png)

After disabling the adaptive mode **the line under the time still shows the adapted duration** but the timer will work as intended.

Yesterday it was the first time I used OnlyT in an actual meeting so I sort of panicked and restarted OnlyT.

**Comment by TizianoNoto** [comment]
I forgot to mention I'm running version 1.3.0.1.
Anyway, I think I found what the problem was.

When starting the timer with one-way adaptive mode enabled the application will show the user it picks the shortest duration:
![immagine](https://user-images.githubusercontent.com/42154401/77684715-9d202e00-6f9a-11ea-9b8a-5640898bf703.png)

After disabling the adaptive mode **the line under the time still shows the adapted duration** but the timer will work as intended.

Yesterday it was the first time I used OnlyT in an actual meeting so I sort of panicked and restarted it.
